packages/react-core/src/components/Nav/Nav.tsx

```diff
@@ -55,9 +55,15 @@ export const NavContext = React.createContext<{
   onToggle?: (event: React.MouseEvent<HTMLButtonElement>, groupId: number | string, expanded: boolean) => void;
   updateIsScrollable?: (isScrollable: boolean) => void;
   isHorizontal?: boolean;
+  flyoutRef?: React.Ref<HTMLLIElement>;
+  setFlyoutRef?: (ref: React.Ref<HTMLLIElement>) => void;
+  disableHover?: boolean;
```

Comment:
Maybe `isHoverDisabled` for naming consistency.

Reply:
Checking again, this prop is used internally by Menu to disable mouseover when some keyboard behavior kicks in. I copied over the whole context initially for Nav, but I actually can probably remove this prop for now since the value never changes.